example/lint.sh

```diff
@@ -46,6 +46,9 @@ args+=(
 	# See https://github.com/aspect-build/rules_ts/pull/574#issuecomment-2073632879
 	"--norun_validations"
 	"--build_event_json_file=$buildevents"
+	# Required for the buf allow_comment_ignores option to work properly
+	# See https://github.com/bufbuild/rules_buf/issues/64#issuecomment-2125324929
+  "--experimental_proto_descriptor_sets_include_source_info"
```

Comment:
@comius do you happen to know if anyone is working on making this flag non-experimental?

Reply:
This was the original issue added the experimental flag, and seems like it was automatically closed last year. https://github.com/bazelbuild/bazel/issues/9337